manifests/pulp_client.pp

```diff
@@ -0,0 +1,48 @@
+# Pulp Client Certs
+class certs::pulp_client (
+  $hostname                          = $::certs::node_fqdn,
+  $generate                          = $::certs::generate,
+  $regenerate                        = $::certs::regenerate,
+  $deploy                            = $::certs::deploy,
+
+  $common_name                       = $::certs::common_name
```

Comment:
Could you reduce the horizontal space between the parameters and their values here?